### DISCUSSION TITLE
Windows ownership of git repository?

### DISCUSSION THREAD

**Discussion by eggbean** [chrisant996/clink#539] [Q&A] [answered]
I'm setting up another user account and I got this message when starting cmd/clink. I'm assuming this account was from the person who last committed to the repository, but that doesn't fit with how I think git works, so what is this about?

```
Microsoft Windows [Version 10.0.22631.2861]
(c) Microsoft Corporation. All rights reserved.

Clink v1.6.0.3e6b73
Copyright (c) 2012-2018 Martin Ridgers
Portions Copyright (c) 2020-2023 Christopher Antos
https://github.com/chrisant996/clink

fatal: detected dubious ownership in repository at 'F:/Users/webadmin/winfiles/Settings/clink-completions'
'F:/Users/webadmin/winfiles/Settings/clink-completions' is owned by:
        'S-1-5-32-544'
but the current user is:
        'S-1-5-21-1915231470-3847582219-1764943238-1007'
To add an exception for this directory, call:

        git config --global --add safe.directory F:/Users/webadmin/winfiles/Settings/clink-completions

```

**Comment by chrisant996** [comment, marked as answer]
This isn't related to Clink or to the repos.

It's purely about local file ownership on the local machine.

An internet search for "`dubious ownership in repository`" finds lots of information that will help you understand and resolve the problem.

Also [Security identifiers](https://learn.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-identifiers) in Windows documentation can help make sense of security identifiers like `S-1-5-32-544`.

And [Highlights from Git 2.36](https://github.blog/2022-04-18-highlights-from-git-2-36/#stricter-repository-ownership-checks).
